### PR TITLE
LPS-109350 As a Marketer, I want to have my home (default) page created as a content page so that I can benefit from latest page fragment features and the creation flexibility it provides

### PR DESCRIPTION
**Pull request by jkappler**

### CODE REVIEW

**Comment by jkappler** [comment]
ci:test:sf

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by david-gutierrez-mesa** [comment]
Just started reviewing :)

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/SessionTimeout.testcase [hunk 1]
```diff
@@ -55,7 +55,7 @@ definition {
 		AssertElementPresent(locator1 = "UserBar#USER_SIGN_IN");
 
 		AssertTextEquals(
-			locator1 = "HelloWorld#PORTLET_TITLE",
+			locator1 = "//h1[contains(@class,'fragment-heading-text-colored')]/div",
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/SessionTimeout.testcase]: We should not use here the XPath directly. Please add it to HelloWorld.path file with a |

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/security/usecase/SecurityUsecase.testcase [hunk 2]
```diff
@@ -198,11 +198,9 @@ out.println(PortalUtil.escapeRedirect("/"));
 			locator1 = "Home#PAGE");
 
 		AssertTextEquals(
-			locator1 = "HelloWorld#PORTLET_TITLE",
+			locator1 = "//h1[contains(@class,'fragment-heading-text-colored')]/div",
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/security/usecase/SecurityUsecase.testcase]: We should not use here the XPath directly. Please add it to HelloWorld.path file with a |

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPages.testcase [hunk 3]
```diff
@@ -315,8 +315,6 @@ definition {
 			menuItem = "Configure",
 			pageName = "Home");
 
-		Page.viewSelectedLayoutColumn(columnName = "1 Column");
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPages.testcase:318]: Why did we remove this here?

File: portal-web/test/functional/com/liferay/portalweb/macros/Smoke.macro [hunk 4]
```diff
@@ -128,13 +128,20 @@ definition {
...
+				locator1 = "HelloWorld#PORTLET_TITLE",
+				value1 = "Hello World");
+
+			AssertTextEquals.assertPartialText(
+				locator1 = "HelloWorld#PORTLET_CONTENT",
+				value1 = "Welcome to Liferay");
+		}
+		else {
+			AssertTextEquals(
+				locator1 = "//h1[contains(@class,'fragment-heading-text-colored')]/div",
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/macros/Smoke.macro]: Please don't use XPath here. Crete it in the appropriated .path file

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPages.testcase (hunk 3 above)
Comment (pavel-savinov) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPages.testcase:318]: Layout selector doesn't exist for Content Pages configuration, only for Widget Pages.

Comment (david-gutierrez-mesa) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPages.testcase:318]: but here the file is WidgetPages.test case

Comment (pavel-savinov) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPages.testcase:318]: It tests Home page configuration, and Home is not a Widget page anymore. Maybe it makes sense to create some Widget page explicitly in this test case then.

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:test:sf

**Comment by david-gutierrez-mesa** [comment]
ci:test:echo

**Comment by david-gutierrez-mesa** [comment]
Just started reviewing :)

**Pull request closed without merging**